src/layout/SubForm/SubFormComponent.tsx

```diff
@@ -105,6 +113,8 @@ function SubFormTableRow({
   const url = getDataModelUrl(instance.id, id, true);
   const { isFetching, data, error } = useFormDataQuery(url);
   const { langAsString } = useLanguage();
+
+  const deleteEntryMutation = useDeleteEntryMutation(id); // TODO: this should be the id of the sub form
```

Comment:
I believe this is the actual data element id. It refers back to `IData.id`, originating from `const dataElements = useStrictInstanceData().data.filter((d) => d.dataType === dataType);`